packages/storage/amplify_storage_s3/example/integration_test/utils/sign_in_new_user.dart

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
+import 'package:amplify_flutter/amplify_flutter.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'tear_down.dart';
+
+/// Creates a new user and signs them in.
+///
+/// Returns the users identityId.
+///
+/// A tear down will be added to delete the user after the test completes.
+Future<String> signInNewUser() async {
+  addTearDownCurrentUser();
+  final username = 'test-user-1-${uuid()}';
+  const password = 'TestUser1!';
```

Comment:
fix: we should be generating the username and password, we have a util for this: https://github.com/aws-amplify/amplify-flutter/blob/main/packages/test/amplify_test/lib/src/mock_data.dart#L60-L65

Reply:
Updated